src/EdFi.AnalyticsMiddleTier.DataStandard2/Base/MSSQL/0030-View-SectionDim-for-Student-Engagement.sql

```diff
@@ -0,0 +1,69 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
```

Comment:
This screenshot shows the changes, when compared to the last alteration of `SectionDim`.

![image](https://user-images.githubusercontent.com/9324390/125685141-f19d1809-3e6e-44a2-ab42-361ef1991dfe.png)

The join to `CourseOffering` did not use all of the fields in the primary key. Adding the two additional fields allowed me to remove `DISTINCT` from the query.

On the first line, `SchoolId` is not nullable, therefore there is no need for `COALESCE`.

Note there are three new fields: `LocalEducationAgencyKey`, `CourseTitle`, and `SessionKey`.

Finally, there is a bug that I have not fixed, and will create a separate ticket for. `CourseTitle as SessionName` is not appropriate, as they are very different concepts. The correct column from the `Session` table was used in the DS31 and DS32 views.